doc/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx>=1.4
+sphinx_gallery
```

Comment:
isn't this adding a dependency? since only developers will be building the docs, we shouldn't add it here ...